testing/tests/DevExpress.ui/defaultOptions.tests.js

```diff
@@ -720,21 +708,6 @@ testComponentDefaults(LoadIndicator,
     }
 );
 
-testComponentDefaults(LoadIndicator,
```

Comment:
Should we rework this behavior dependent on `platform == ios`?
/cc @DokaRus

Reply:
That won't be necessary. This test checks that the options are set to the correct value depending on the topic. In this PR, this logic is removed [here](https://github.com/DevExpress/DevExtreme/pull/13459/files/337dcfd213676609212bb9edccace54f85a45926#diff-f13e54a3d16972c74fc154d4278892d5L80).

Reply:
>In this PR, this logic is removed here.

I know. I meant, should we save this behavior on iOS devices for all themes (Generic especially)?